tests/gtests/test_eltwise.cpp

```diff
@@ -370,19 +428,64 @@ TEST_P(eltwise_test_bfloat16, TestsEltwise)
     EXPAND(PARAMS(eltwise_bounded_relu, __VA_ARGS__)), \
     EXPAND(PARAMS(eltwise_logistic, __VA_ARGS__))
 
+#if defined(DO_TEST_BF16) && DO_TEST_BF16
+    #define CPU_INST_TEST_CASE_BF16(str, ...) _CPU_INST_TEST_CASE(str, bf16, __VA_ARGS__);
+    #define INST_TEST_CASE_BF16(str, ...) _INST_TEST_CASE(str, bf16, __VA_ARGS__);
+#else
+    #define CPU_INST_TEST_CASE_BF16(str, ...)
+    #define INST_TEST_CASE_BF16(str, ...)
+#endif
+
+#define _CPU_INST_TEST_CASE(str, data_t, ...) CPU_INSTANTIATE_TEST_SUITE_P( \
+        str##_##data_t, eltwise_test_##data_t, ::testing::Values(__VA_ARGS__))
+
+#define _INST_TEST_CASE(str, data_t, ...) INSTANTIATE_TEST_SUITE_P_( \
```

Comment:
should these macros `_CPU_INST_TEST_CASE` and `_INST_TEST_CASE` be moved before line 431 ?